release notes/v0.36.0.md

```diff
@@ -0,0 +1,136 @@
+```javascript
+import exec from "k6/execution";
+
+export default function() {
+  // Note that you can abort with a specific message too
+  exec.test.abort("this is the reason");
+}
+
+export function teardown() {
+  console.log("teardown will still be called after test.abort()");
+}
+```
+
+Or during the VU initialization:
+```javascript
+import exec from "k6/execution";
+
+// This won't fail on initial parsing of the script, but on VU initialization.
+if (__VU == 1) {
+  exec.test.abort();
+}
+
+export default function() {}
+```
+
+### k6 inspect extended output ([#2279](https://github.com/grafana/k6/pull/2279))
+
+Following [#2279](https://github.com/grafana/k6/pull/2279), the `k6 inspect` command now supports an `--execution-requirements` flag. When used, the command's output will include fields related to the execution requirements, by deriving k6's configuration from the execution context, and including  the `maxVUs` and `totalDuration` fields to the output.
+
+
+### Forcing HTTP/1 protocol ([#2222](https://github.com/grafana/k6/pull/2222))
+
+Thanks to the work of [@sjordhani22](https://github.com/sjordhani22), [#2222](https://github.com/grafana/k6/pull/2222) made it possible to force k6 to use version 1.1 of the protocol when firing HTTP requests. 
+
+It can be done by setting the `http2client=0` value in the `GODEBUG` environment variable:
+
+```
+GODEBUG=http2client=0 k6 run testscript.js
+```
+
+**Nota Bene**: the usage of the `GODEBUG` variable is considered temporary, and expected to change in the future. If you start using this feature, keep an eye out for potential future changes.
+
+## Extensions
+
+`v0.36.0` marks the switch of some of our internal modules to [a new JavaScript extension API](https://k6.io/docs/extensions/guides/create-an-extension/#advanced-javascript-extension). We expect this change to make the process of developing internal JavaScript modules and advanced JavaScript extensions easier and more streamlined in the future. Although this switch to a new API does not introduce breaking changes for existing extensions yet, we anticipate deprecating the current extension API at an undecided point in the future.
+
+For more details, see: [#2243](https://github.com/grafana/k6/pull/2243), [#2241](https://github.com/grafana/k6/pull/2241), [#2239](https://github.com/grafana/k6/pull/2239), [#2242](https://github.com/grafana/k6/pull/2242), [#2226](https://github.com/grafana/k6/pull/2226), and [#2232](https://github.com/grafana/k6/pull/2232).
+
+## Breaking changes
+
+### Restricting file opening to init context
+
+Opening files is now restricted to the init context of the VU that's initialized to get the options from the script. While it was somewhat possible to open files during other stages of the test in the past, it was [unreliable](https://github.com/grafana/k6/issues/1771). [#2314](https://github.com/grafana/k6/pull/2314) ensures that k6 would now throw an error in a similar scenario.
```

Comment:
```suggestion
Opening files is now restricted to the init context of the VU that's initialized to get the options from the script(__VU==0). While it was somewhat possible to open files *only* in other VUs (for example `__VU == 2`) in the past, it was [unreliable](https://github.com/grafana/k6/issues/1771). [#2314](https://github.com/grafana/k6/pull/2314) ensures that k6 would now throw an error in a similar scenario. This still means that you can open files only for some VUs, but you need to have opened all of those files in the initial VU (__VU==0).
```

Maybe it will be better to have another example like 
```
let file;
if (__VU == 0) {
  open("./file1.bin")
  open("./file2.bin")
} else if (__VU % 2 == 0) {
  file = open("./file1.bin")
} else {
  file = open("./file2.bin")
}
export default () => {
  // use file for something
}
```